src/main/java/com/cloudbees/jenkins/support/impl/RunningJobs.java

```diff
@@ -0,0 +1,53 @@
+package com.cloudbees.jenkins.support.impl;
+
+import com.cloudbees.jenkins.support.api.Component;
+import com.cloudbees.jenkins.support.api.Container;
+import com.cloudbees.jenkins.support.api.PrefilteredPrintedContent;
+import com.cloudbees.jenkins.support.filter.ContentFilter;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.Extension;
+import hudson.security.Permission;
+import jenkins.model.Jenkins;
+import org.jenkinsci.plugins.workflow.support.steps.ExecutorStepExecution;
+
+import java.io.PrintWriter;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Extension
+public class RunningJobs extends Component {
+
+    @NonNull
+    @Override
+    public Set<Permission> getRequiredPermissions() {
+        return Collections.singleton(Jenkins.ADMINISTER);
+    }
+
+    @Override
+    @NonNull
+    public String getDisplayName() {
+        return "Running Jobs";
+    }
+
+    @Override
+    public void addContents(@NonNull Container result) {
+        result.add(
+            new PrefilteredPrintedContent("nodes/master/running-jobs.txt") {
+                @Override
+                protected void printTo(PrintWriter out, ContentFilter filter) {
+                    Optional.ofNullable(Jenkins.getInstanceOrNull())
+                        .ifPresent(jenkins -> Arrays.stream(jenkins.getComputers())
+                            .flatMap(computer -> computer.getAllExecutors().stream())
+                            .collect(Collectors.toList())
+                            .forEach(executor -> Optional.ofNullable(executor.getCurrentExecutable())
+                                .filter(executable -> !(executable.getParent() instanceof ExecutorStepExecution.PlaceholderTask))
```

Comment:
Would I think suffice to say
```suggestion
                                .filter(executable -> executable.getParent() == executable.getParent().getOwnerTask())
```

which will filter out `PlaceholderTask`s without needing the dependency on `workflow-durable-task-step`—this could become `test`-scoped.

For reasons I do not fully understand, neither `MatrixConfiguration` nor `MavenModule` seem to override this method, so these would not be filtered out.

Reply:
Interesting. I'll see what happens when I add some tests.